tests/chat_completions_detection.rs

```diff
@@ -138,14 +139,10 @@ async fn no_detections() -> Result<(), anyhow::Error> {
 
     // Add chat completions mock
     chat_mocks.mock(|when, then| {
-        when.post()
-            .path(CHAT_COMPLETIONS_ENDPOINT)
-            .json(ChatCompletionsRequest {
-                messages: messages.clone(),
-                model: MODEL_ID.into(),
-                stream: false,
-                ..Default::default()
-            });
+        when.post().path(CHAT_COMPLETIONS_ENDPOINT).json(json!({
+            "model": MODEL_ID,
+            "messages": messages.clone(),
+        }));
```

Comment:
If you're using `json!()`, most likely you shouldn't need clones.
```suggestion
        when.post().path(CHAT_COMPLETIONS_ENDPOINT).json(json!({
            "model": MODEL_ID,
            "messages": messages,
        }));
```

Reply:
If you agree with this, please do the same for all other tests you've changed.